php/PhpLexer.g4

```diff
@@ -111,6 +111,8 @@ MultiLineComment:   '/*' .*? '*/' -> channel(PhpComments);
 SingleLineComment:  '//' -> channel(SkipChannel), pushMode(SingleLineCommentMode);
 ShellStyleComment:  '#' -> channel(SkipChannel), pushMode(SingleLineCommentMode);
 
+AttributeStart:     '#[';
```

Comment:
It seems like spaces are allowed between `#` and `[`, see https://3v4l.org/5ZYsq

Reply:
No, `#` is start of comment, so `# [Attribute]` will parse. But it's not interpreted as attribute, can't read by getAttributes() reflection API. see https://3v4l.org/7FPRr